packages/core/src/options.ts

```diff
@@ -44,16 +44,18 @@ export interface Emoji {
 	embedEmoji?: boolean;
 }
 
-const globalAvatars: Avatars = window.$discordMessage?.avatars ?? ({} as Avatars);
+const serverSideSafeWindow = window ?? globalThis;
```

Comment:
@TheBv I hope this fixes the NextJS issue but can you confirm?

Reply:
Tested it. We need to check for `typeof window !== undefined` though because that's the only way next.js can properly check if window is defined or not (due to some server-side/client-side stuff).